compliance/scripts/compliance_cli.py

```diff
@@ -108,6 +108,8 @@ def _init_arguments(self):
         )
 
     def _validate_arguments(self, args):
+        if 'stdout' not in args.notify:
```

Comment:
What about not having stdout in the list the user can choose from? Should update the help at line 94 to say stdout is always used.

Something for another time, if we were on py3.8 we could do something like:

```
import argparse
parser = argparse.ArgumentParser(prog='game.py')
parser.add_argument('--notify', choices=['gh_issue', 'slack', 'pagerduty'], nargs='*', action='extend')
parser.parse_args('--notify gh_issue pagerduty --notify slack'.split())
```

Reply:
I updated the help to exclude "stdout" from the valid list of values and added a note saying that the stdout notifier will always run.  There's nothing stopping someone from specifying "stdout" as a notifier which lends to backwards compatibility.  I think that's a good thing.